plugins/contact-resources/src/components/EditPerson.svelte

```diff
@@ -54,18 +54,6 @@
 
   const dispatch = createEventDispatcher()
 
-  function firstNameChange () {
```

Comment:
We still need to edit contacts, we should disable editing for Employees only.

Reply:
fixed